.github/workflows/prepare_release.yml

```diff
@@ -18,7 +18,9 @@ jobs:
         with:
           fetch-depth: 2
       - name: Set up Git user
-        uses: fregante/setup-git-user@v2
+        run: |
+          git config --global user.name "pulumi-kotlin[bot]"
+          git config --global user.email "138587440+pulumi-kotlin[bot]@users.noreply.github.com"
```

Comment:
Would it be possible to add a custom email for this GitHub app and use it here?

Reply:
It doesn't seem to be possible.